modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,37 @@
+[id="enabling-ang-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+
+.Procedure
+. To give {product-short} access to GitHub, link:{linkgettingstartedguide}#configuring-github-integration[configure GitHub integration].
+
+. The Bulk Import plugins are installed but disabled by default.
+To enable the `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic` and `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import` plugins,
+edit your `dynamic-plugins.yaml` with following content.
+See link:{plugins-configure-book-url}[{plugins-configure-book-title}].
++
+.`dynamic-plugins.yaml` fragment
+[source,yaml]
+----
+global:
+  dynamic:
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic
+        disabled: false
+      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import
+        disabled: false
+----
+
+. Only {product-short} administrators or users with the `bulk.import` permission can use the Bulk Import feature.
+Configure the required `bulk.import` RBAC permission for the users that are not administrators.
+See link:{authorization-book-url}#ref-rbac-permission-policies_title-authorization[Permission policies in Red Hat Developer Hub].
```

Comment:
I guess an example of external policy file with the bulk-import permission might be helpful here:
```yaml
kind: ConfigMap
apiVersion: v1
metadata:
  name: rbac-policy
  namespace: rhdh
data:
  rbac-policy.csv: |
    p, role:default/bulk-import, bulk.import, use, allow
    p, role:default/bulk-import, catalog-entity.read, read, allow
    p, role:default/bulk-import, catalog.entity.create, create, allow
    g, user:default/<YOUR_USER>, role:default/bulk-import
```

Reply:
```suggestion
See link:{authorization-book-url}#ref-rbac-permission-policies_title-authorization[Permission policies in Red Hat Developer Hub].
+
.`rbac-policy.csv` fragment
[source,csv,subs="+quotes"]
----
p, role:default/bulk-import, bulk.import, use, allow
p, role:default/bulk-import, catalog-entity.read, read, allow
p, role:default/bulk-import, catalog.entity.create, create, allow
g, user:default/__<your_user>__, role:default/bulk-import
----
```